test/kubernetes_unittest.cc

```diff
@@ -1258,7 +1275,7 @@ bool WaitForNewerCollectionTimestamp(const MetadataStore& store,
 // to the store.
 void TestNodes(testing::FakeServer& server, MetadataStore& store,
                const std::string& nodes_watch_path) {
-  server.WaitForOneStreamWatcher(nodes_watch_path, time::seconds(3));
+  server.WaitForMinTotalConnections(nodes_watch_path, 1, time::seconds(3));
```

Comment:
Would be nice to have a local `const auto timeout = time::seconds(3);` here and in other functions.